src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double leftSide;
+    private double downSide;
+
+    public Rectangle() {
+    }
+
+    public Rectangle(double leftSide, double downSide) {
+        this.typeOfFigure = TypeOfFigure.RECTANGLE;
+        this.area = getQuadrangularArea(leftSide, downSide);
+        this.leftSide = leftSide;
+        this.downSide = downSide;
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables for 'leftSide' and 'downSide' before passing them to the constructor for better code readability.